docs/audio.rst

```diff
@@ -129,8 +129,8 @@ Sound Effects **V2**
     :param duration: Duration of the sound (ms), default: ``500``
     :param vol_start: Start volume value, range 0-255, default: ``255``
     :param vol_end: End volume value, range 0-255, default: ``0``
-    :param wave: Type of wave shape, one of these values: ``WAVE_SINE``,
-        ``WAVE_SAWTOOTH``, ``WAVE_TRIANGLE``, ``WAVE_SQUARE``,
+    :param waveform: Type of waveform shape, one of these values:
```

Comment:
A suggestion by @microbit-giles

Reply:
I suggest we then also change the constants to `WAVEFORM_SAWTOOTH` etc.

Reply:
I think consistency is important here.